### ISSUE TITLE
Remote-Containers: Add Development Container Configuration Files stalls on /.yarn/cache

### ISSUE THREAD

**Issue by yujuncho7** [microsoft/vscode-remote-release#4744]
- VSCode Version: 1.54.3 (Also reproduced with 1.55.0-insider)
- Local OS Version: Windows 10.0.19041 Build 19041
- Remote OS Version: Ubuntu 20.04
- Remote Extension/Connection Type: WSL

Steps to Reproduce:

1. Open a project folder that contains `/.yarn/cache` (Note, I'm using Yarn 2)
2. Run `Remote-Containers: Add Development Container Configuration Files`
3. The list of container definitions is empty. `Remote-Containers: Show Log` seems to indicate that the command to create a config file resulted in reviewing `/.yarn/cache`

![image](https://user-images.githubusercontent.com/4506629/112199137-157bf800-8bcb-11eb-8751-a7c23847b10e.png)

4. I haven't quite figured out how to reproduce this step reliably yet. When I attempt to re-run the command in step 2, there were instances where VS Code eventually told me that the commands for Remote-Containers couldn't be found. In my most recent attempt, I tried to re-run the commands, disabled my extensions, turned on Remote - WSL, attempted to re-connect, and the connection failed. Restarting my computer fixed this for me.

![image](https://user-images.githubusercontent.com/4506629/112200105-20835800-8bcc-11eb-88a7-bb15bbf966a8.png)

**Comment by chrmarti** [comment]
Are the `*.zip` paths directories or files? (It shouldn't run `ls` on files.)

**Comment by yujuncho7** [comment]
@chrmarti `test -f` says the `*.zip` paths are files.

**Comment by chrmarti** [comment]
Could you attach the full log?

**Comment by yujuncho7** [comment]
I don't believe the attached log is the same as the one that I screenshot, but at the bottom of this attached log, it shows a similar output of running `ls` on `*.zip` paths.

[remoteContainers-2021-03-24T18-46-43.646Z.log](https://github.com/microsoft/vscode-remote-release/files/6206459/remoteContainers-2021-03-24T18-46-43.646Z.log)

**Comment by connor4312** [comment]
@yujuncho7 please try this out in [Insiders](https://code.visualstudio.com/insiders/) and let us know if it's resolved for you.
